app/action_creators/filter.js

```diff
@@ -0,0 +1,30 @@
+import {
+  SET_FILTER_ENTERED,
+  SET_FILTER_ACTIVATED,
+} from '../action_types';
+
+import { getEnteredFilter } from '../reducers';
+import { fetchCases } from './cases';
+
+const minSearchTermLength = 3;
```

Comment:
not sure what you have in mind -- will merge now. curious to hear the alternative tomorrow.